cmd/kubelet/app/server.go

```diff
@@ -221,6 +221,10 @@ is checked every 20 seconds (also configurable with a flag).`,
 				}
 			}
 
+			if kubeletFlags.DisableCgroupV1Support && !isCgroup2UnifiedMode() {
```

Comment:
I don't think we should check both. the flags should get converted to the config, no? we can just check below if so